epersons.md

```diff
@@ -204,6 +207,47 @@ Status codes:
 * 403 Forbidden - if you are not logged in with sufficient permissions
 * 422 Unprocessable Entity - if the email address was omitted or already exists
 
+## Create new EPerson based on registration token
+**POST /api/eperson/epersons?token=<:token>**
+
+To create a new EPerson, perform a post with the JSON below to the epersons endpoint while including a token.
+The token will be sent via Email from the [Create new EPerson registration](epersonregistrations.md#create-new-eperson-registration).
+
+```json
+{
+  "metadata": {
+    "eperson.firstname": [
+      {
+        "value": "John",
+        "language": null,
+        "authority": "",
+        "confidence": -1
+      }
+    ],
+    "eperson.lastname": [
+      {
+        "value": "Doe",
+        "language": null,
+        "authority": "",
+        "confidence": -1
+      }
+    ]
+  },
+  "canLogIn": true,
+  "requireCertificate": false,
+  "type": "eperson"
+}
+```
+
+The "eperson.firstname" and "eperson.lastname" metadata are mandatory. The phone number, language, … are optional metadata.  
+The email property can be set, but would need to be identical to the value from the registration.  
+The selfRegistered property can be set, but would need to be true
+
+Status codes:
+* 201 Created - if the operation succeed
+* 401 Unauthorized - if the token doesn't allow you to create this account
+* 422 Unprocessable Entity - if the email address didn't match the token or already exists
```

Comment:
We likely also need an error code for an expired token, as I believe they have an expiration date (maybe a 400 Bad Request?).  I'm also wondering if an email address not matching should be a 400 status code instead of 422.  But, I don't feel strongly enough either way here.

Reply:
I've moved it to 200.
Expired tokens don't exist anymore (not sure why, but it has been removed from Java a long time ago). But I included it in case it would be reinstated in the future